src/test/docker-compose.yml

```diff
@@ -9,9 +9,10 @@ services:
       - 9229:9229
 
   nivio:
-    image: bonndan/nivio:latest
+    image: dedica/nivio:latest
     environment:
-      DEMO: 1
+      DEMO: ${DEMO}
+      SEED: ${SEED}
```

Comment:
By this, we could use this docker-compose.yml for both the demo and the seed config as described in the manual. @bonndan: were these docker-compose.yml files even intended for this purpose?

Reply:
No, you have found an ancient relict. However, it could work that way. Using both seed and demo is now supported.

Reply:
Okay, so we could leave it like that? 

And on line 6 we are referencing to the image ` bonndan/dld4e-headless:latest` in this docker-compose.yml. This is your private Docker Hub user, right? Do you intend on keeping that image there?

Reply:
OMG I was not aware the documentation is so outdated. Sorry for the confusion. I have pushed some cleanups.